geoprocessing/src/migrations/geoprocessing/1621431520169-AddBboxToAdminAreasAndProjects.ts

```diff
@@ -0,0 +1,78 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+export class AddBboxToAdminAreasAndProjects1621431520169
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    -----------------------------------------
+    -- Generates the new column we need
+    -----------------------------------------
+    ALTER TABLE admin_regions ADD COLUMN bbox jsonb;
```

Comment:
That one is entending from the country View, so only defining it in the country should be more than  enough

Reply:
@aagm you don't have to expose it to frontend (i.e. don't put Api decorators) but I believe we should keep the code definitions really close (the same) as they are in the database - it will help with both generating migrations automatically AND keep the usage clean (right now, if you would like to use bbox from within the code - you can't; and in few months one will need to figure out that the column is there by reviewing every single migration where this table is present)